README.md

```diff
@@ -1,6 +1,15 @@
 # obscuritext
 A Python script to obscure free text into anonymized data by hashing each word. These hashes can still be used for statistical modeling but have no identifying information and are completely unreadable.
```

Comment:
"transform free text into de-identified data"

Reply:
"but have no identifying information and are completely unreadable" -> "while effectively making the text unreadable"